samples/samples/admin/samples.py

```diff
@@ -103,3 +181,525 @@ def create_database_with_default_leader(instance_id, database_id, default_leader
 
 
 # [END spanner_create_database_with_default_leader]
+
+
+# [START spanner_create_database_with_encryption_key]
+def create_database_with_encryption_key(instance_id, database_id, kms_key_name):
+    """Creates a database with tables using a Customer Managed Encryption Key (CMEK)."""
+    from google.cloud.spanner_admin_database_v1 import EncryptionConfig
+
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+
+    request = spanner_database_admin.CreateDatabaseRequest(
+        parent=instance.name,
+        create_statement=f"CREATE DATABASE `{database_id}`",
+        extra_statements=[
+            """CREATE TABLE Singers (
+            SingerId     INT64 NOT NULL,
+            FirstName    STRING(1024),
+            LastName     STRING(1024),
+            SingerInfo   BYTES(MAX)
+        ) PRIMARY KEY (SingerId)""",
+            """CREATE TABLE Albums (
+            SingerId     INT64 NOT NULL,
+            AlbumId      INT64 NOT NULL,
+            AlbumTitle   STRING(MAX)
+        ) PRIMARY KEY (SingerId, AlbumId),
+        INTERLEAVE IN PARENT Singers ON DELETE CASCADE""",
+        ],
+        encryption_config=EncryptionConfig(kms_key_name=kms_key_name),
+    )
+
+    operation = spanner_client.database_admin_api.create_database(request=request)
+
+    print("Waiting for operation to complete...")
+    database = operation.result(OPERATION_TIMEOUT_SECONDS)
+
+    print(
+        "Database {} created with encryption key {}".format(
+            database.name, database.encryption_config.kms_key_name
+        )
+    )
+
+
+# [END spanner_create_database_with_encryption_key]
+
+
+def add_and_drop_database_roles(instance_id, database_id):
+    """Showcases how to manage a user defined database role."""
+    # [START spanner_add_and_drop_database_role]
+    # instance_id = "your-spanner-instance"
+    # database_id = "your-spanner-db-id"
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+    database = instance.database(database_id)
+
+    role_parent = "new_parent"
+    role_child = "new_child"
+
+    request = spanner_database_admin.UpdateDatabaseDdlRequest(
```

Comment:
nit: can we add import of spanner_database_admin within this function, similar to sample in `create_database_with_encryption_key`. This will help customers understand where is it taken from.

Reply:
I will defer it to another PR since we need to refactor all the samples to follow this.

I want this PR to simple lift and shift from handwritten to autogenerated one, happy to discuss more offline.

Reply:
I am fine with this if we can do this before we make these samples public.
Otherwise customers would not understand what `spanner_database_admin ` is referring to.

Reply:
Updated all the samples to have import